ecs-cli/modules/commands/compose/compose_command.go

```diff
@@ -79,10 +80,10 @@ func composeFlags() []cli.Flag {
 			Name:  command.VerboseFlag + ",debug",
 			Usage: "Increase the verbosity of command output to aid in diagnostics.",
 		},
-		cli.StringFlag{
+		cli.StringSliceFlag{
 			Name:   command.ComposeFileNameFlag + ",f",
-			Usage:  "Specifies the Docker compose file to use. Defaults to " + composeFileNameDefaultValue + " file.",
-			Value:  composeFileNameDefaultValue,
+			Usage:  "Specifies one or more Docker compose files to use. Defaults to " + composeFileNameDefaultValue + " file, and an optional " + composeOverrideFileNameDefaultValue + " file.",
+			Value:  &cli.StringSlice{},
```

Comment:
Or actually may be not good idea... I made the change and now it ignores the files specified by --file and only looks for docker-compose.yml and docker-compose.override.yml

```
$ ecs-cli compose -f hello-ecs-mysql.yml -f hello-ecs-wordpress.yml up
ERRO[0000] Failed to find the compose file: docker-compose.yml 
ERRO[0000] Unable to open ECS Compose Project            error="open docker-compose.yml: no such file or directory"
FATA[0000] Unable to create and read ECS Compose Project  error="open docker-compose.yml: no such file or directory"
186590d48925:test1 wppttt$ 
```